src/commands/case_management.py

```diff
@@ -477,15 +475,12 @@ async def cmd_case_management_quiet(ctx: Context):
         await ctx.reply("Got no information yet")
         return
 
-    delta = relativedelta(datetime.now(tz=timezone.utc), ctx.bot.board.last_case_datetime)
-    hour_part = ""
-    if delta.hours:
-        hour_plural = 's' if delta.hours > 1 else ''
-        hour_part = f"{delta.hours} hour{hour_plural} and "
-    minute_plural = 's' if delta.minutes != 1 else ''
-    await ctx.reply(
-        f"The last case was created {hour_part}{delta.minutes} minute{minute_plural} ago."
-    )
+    human_delta = pendulum.now().diff_for_humans(ctx.bot.board.last_case_datetime)
```

Comment:
Same from your own comment at line 587:
[style] We should be consistent, is it pendulum.now() or pendulum.now(pendulum.tz.UTC).
I believe the two forms are consistent, since pendulum datetimes are always TZ aware, defaulting to the current locale.
Logically, either don't specify a timezone or always use UTC. Since all pendulum datetimes are tz-aware, it appears to be a stylistic choice.

Reply:
to heck with it. `pendulum.now()` it is.

since its TZ aware it makes no difference.

Reply:
Unless we specifically are using it for rendering, we can simply:tm: omit the timezone.

For rendering we can cast it to UTC at that time, which is a cheap operation.